src/connections/SitePolygons.ts

```diff
@@ -31,17 +32,18 @@ const ENTITY_QUERY_KEYS: Record<string, string> = {
   sites: "siteId[]"
 };
 
-export type SitePolygonIndexConnection<SitePolygonDto> = {
-  sitePolygons?: SitePolygonDto[];
+export type SitePolygonIndexConnection<SitePolygonLightDto> = {
+  sitePolygons?: SitePolygonLightDto[];
   meta?: ResponseMeta["page"];
   fetchFailure?: PendingErrorState | null;
 };
 
 const sitePolygonQueryParams = (props: SitePolygonIndexConnectionProps) => {
   const queryKey = ENTITY_QUERY_KEYS[props.entityName];
-  const queryParams: Record<string, string | number | null | undefined> = {
+  const queryParams: Record<string, string | number | boolean | null | undefined> = {
     "page[number]": props.pageNumber ?? 1,
-    "page[size]": props.pageSize ?? 10
+    "page[size]": props.pageSize ?? 10,
+    lightResource: props.lightResource ?? false
```

Comment:
Isn't the FE always going to request the light DTO, or do I have that wrong? If that's right, you can remove it from the props everywhere and just hardcode to `true` here.